felwithea/#Galrun_Starforge.lua

```diff
@@ -1,7 +1,14 @@
+-- epic expedition version of Struggles within the Progeny
+-- Game Update Notes: April 19, 2017
+-- Valdoon Paladin Epic 2.0 (Raid) - Lowered the minimum number of players necessary to request this raid from 10 to 6.
+local paladin_minimum_players = 10
+local paladin_expedition = { "The Asylum of Invoked Stone", paladin_minimum_players, 54 }
```

Comment:
The only thing I don't like about the array context configuration is the readability of array arguments, I like that the variable was named here because it makes it readable. It seems like a small thing but it is a simple detail I just want to make sure we consider before finalizing everything

If it was more object declared it would remove the need for needing to make locals, not sure if this is a thing in LUA though

Example

```
{
	"dz_name": "The Asylum of Invoked Stone",
	"minimum_players": 10,
	"maximum_players": 54
}
```

Reply:
I can look into implementing this as a hash instead of an array. I've been bitten by this myself where I forgot an argument and received an ambiguous casting error, so I agree

Reply:
That would be really awesome and put a nice polish on everything